bbs/xferovl1.cpp

```diff
@@ -480,7 +480,7 @@ void tag_files(bool& need_title) {
       bout.clear_lines_listed();
       a()->filelist.clear();
       need_title = true;
-      bout.cls();
+      //bout.cls();
```

Comment:
why?

Reply:
oh.. in transfer section, when in non-ANSI mode, when you L)ist files, you get the File Tagging prompt and Q)uitting didn't clear the screen, whereas C)ontinue, SPACE or RETURN would. It wasn't a consistent experience. I did tests with it and not clearing the screen made more sense. an option i was thinking of was to make  bout.cls() behavior in spots like these follow the D)efaults S) Clear screen between messages switch. Clear the screen if that toggle is on.

Reply:
that sounds great to have it follow an option like that.